package-lock.json

```diff
@@ -2013,16 +2162,37 @@
             "pretty-hrtime": "^1.0.0",
             "replace-homedir": "^1.0.0",
             "semver-greatest-satisfied-range": "^1.1.0",
-            "v8flags": "^3.2.0"
-          },
-          "dependencies": {
-            "yargs": {
-              "version": "13.3.2"
-            }
+            "v8flags": "^3.2.0",
+            "yargs": "^7.1.0"
           }
         }
       }
     },
+    "gulp-cli": {
```

Comment:
We probably do need this as we've put gulp-cli in the pakage file but if so we need to update the yargs dependency on it